### PR TITLE
Add gpmimerg dataset

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
Adding GPM IMERG as way to demonstrate titiler-cmr within VEDA.

@j08lue this is working but the collection is [only published in staging](https://staging.openveda.cloud/api/stac/collections/GPM_3IMERGDF), presumably it should be published in prod. 

@anayeaye - would it be a lot of work to publish the collection to prod? It's just a collection record, with no items.

### CODE REVIEW

**Comment by j08lue** [comment]
Hmm, getting 500 right now e.g. on 

https://dev-titiler-cmr.delta-backend.com/tiles/WebMercatorQuad/1/1/1@1x?datetime=2024-10-20T22%3A00%3A00.000Z&resampling=bilinear&variable=precipitation&colormap_name=gnbu&rescale=0%2C46&concept_id=C2723754864-GES_DISC&backend=xarray

https://deploy-preview-462--visex.netlify.app/exploration?datasets=%5B%7B%22id%22%3A%22GPM_3IMERGDF.v07%22%2C%22settings%22%3A%7B%22isVisible%22%3Atrue%2C%22opacity%22%3A100%2C%22analysisMetrics%22%3A%5B%7B%22id%22%3A%22mean%22%2C%22label%22%3A%22Average%22%2C%22chartLabel%22%3A%22Average%22%2C%22themeColor%22%3A%22infographicB%22%7D%2C%7B%22id%22%3A%22std%22%2C%22label%22%3A%22St+Deviation%22%2C%22chartLabel%22%3A%22St+Deviation%22%2C%22themeColor%22%3A%22infographicD%22%7D%5D%2C%22colorMap%22%3A%22gnbu%22%7D%7D%5D&taxonomy=%7B%7D&search=

**Comment by j08lue** [comment]
Ah, so this is how it fails when the requested datetime is not found?

**Comment by abarciauskas-bgse** [comment]
@j08lue yes - I can see that we probably need better error handling here

**Comment by j08lue** [comment]
I see the same request for 2022 works: https://dev-titiler-cmr.delta-backend.com/tiles/WebMercatorQuad/0/0/0@1x?datetime=2022-10-20T00%3A00%3A00.000Z&resampling=bilinear&variable=precipitation&colormap_name=gnbu&rescale=0%2C46&concept_id=C2723754864-GES_DISC&backend=xarray

All we need for this to work in VEDA Dashboard Exploration is probably to set the date range end date in `extent > temporal > interval` (currently that is `null`), right?

https://staging.openveda.cloud/api/stac/collections/GPM_3IMERGDF

**Comment by abarciauskas-bgse** [comment]
@j08lue yes I think we just need to update that interval to end at May 31st, 2024 (for now). I'm going to look at the docs to learn about how this is done now and reach out to the data service team if I need help.

**Comment by abarciauskas-bgse** [comment]
Thanks to help from @anayeaye this is now published in prod (https://openveda.cloud/api/stac/collections/GPM_3IMERGDF), with the current end date added to the metadata, so I have updated the config file accordingly. I think it is good to go - what do you think @j08lue ?

File: datasets/GPM_3IMERGDF.data.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    sourceParams:
+      resampling: bilinear
+      variable: precipitation
+      colormap_name: gnbu
+      rescale:
+        - 0
+        - 46
+      maxzoom: 12
+      concept_id: C2723754864-GES_DISC
+      backend: xarray
```
Comment (j08lue) [inline datasets/GPM_3IMERGDF.data.mdx:35]:
```suggestion
      backend: xarray
      tile_scale: 3
```

I think we should do the same thing as in https://github.com/US-GHG-Center/veda-config-ghg/pull/630 to make the data look its best (crispiest) in demos, even though the user / information value is perhaps limited.

**Review by j08lue — APPROVED** [review]
Works nicely now! Should we use tile scale 3x here, as in https://github.com/US-GHG-Center/veda-config-ghg/pull/630, to make the data look even better? 💄

**Comment by abarciauskas-bgse** [comment]
I did some basic testing and it doesn't appear there is a performance difference between tile_scale 1 vs tile_scale 3:

```
➜ curl -o /dev/null -s -w "%{time_total}\n" https://dev-titiler-cmr.delta-backend.com/tiles/WebMercatorQuad/0/0/0@1x\?datetime\=2024-05-31T07%3A00%3A00.000Z\&resampling\=bilinear\&variable\=precipitation\&colormap_name\=gnbu\&rescale\=0%2C46\&concept_id\=C2723754864-GES_DISC\&backend\=xarray
2.563051
```

```
➜  curl -o /dev/null -s -w "%{time_total}\n" https://dev-titiler-cmr.delta-backend.com/tiles/WebMercatorQuad/0/0/0@3x\?datetime\=2024-05-31T07%3A00%3A00.000Z\&resampling\=bilinear\&variable\=precipitation\&colormap_name\=gnbu\&rescale\=0%2C46\&concept_id\=C2723754864-GES_DISC\&backend\=xarray
2.674560
```

**Review by j08lue — APPROVED** [review]
Looks great!

https://deploy-preview-462--visex.netlify.app/data-catalog/GPM_3IMERGDF.v07

**Review by hanbyul-here — APPROVED** [review]

**Pull request merged**